server/models/Profile.js

```diff
@@ -3,25 +3,36 @@ const mongoose = require("mongoose");
 const profileSchema = new mongoose.Schema({
   firstName: {
     type: String,
+    default: "",
```

Comment:
You need to figure out if this property is required and add the necessary attribute. There's no need to add an empty string. It defaults to a value of `undefined`.